.github/workflows/verifyHybridApp.yml

```diff
@@ -52,10 +63,17 @@ jobs:
         with:
           IS_HYBRID_BUILD: 'true'
 
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1.204.0
+        with:
+          bundler-cache: true
+
+      - name: Install New Expensify Gems
+        run: bundle install
```

Comment:
Pretty sure that `ruby/setup-ruby` runs `bundle install` internally?

Reply:
Can confirm, it runs `bundle install` in `setup-ruby`:

<img width="531" alt="image" src="https://github.com/user-attachments/assets/693894b3-6535-4075-8e26-cdfdf60baf8a" />

Reply:
Nice catch! Will remove